### PR TITLE
Add "Remove Your Club" modal

### LINKED ISSUE #341: Build "remove your club" workflow
Design was done here:

Authenticated user with Club (sidebar): http://invis.io/YC2HKEHQD
Remove club: http://invis.io/6W2HKGLMV
Remove confirmation: http://invis.io/F92HKGSK5

Acceptance test:
 - Club should no longer be accessible in db

### PR DESCRIPTION
**Pull request by toolness**
This resolves #341.

TODOs:
- [x] Add tests
- [x] Use @vazquez's "globe without pin" graphic (@vazquez, where is this?)

### CODE REVIEW

**Comment by vazquez** [comment]
Here it is! https://dl.dropboxusercontent.com/u/43783651/Mozilla/globe-without-pin.svg

File: pages/clubs.jsx [hunk 1]
```diff
@@ -85,6 +85,76 @@ var BottomCTA = React.createClass({
...
+    }
+    this.setState({
+      networkError: !!err,
+      step: err ? this.STEP_CONFIRM : this.STEP_SHOW_RESULT
+    });
+  },
+  render: function() {
+    var content, isFormDisabled;
+
+    if (this.state.step != this.STEP_SHOW_RESULT) {
```
Comment (ScottDowne) [inline pages/clubs.jsx]: !==

File: pages/clubs.jsx [hunk 2]
```diff
@@ -85,6 +85,76 @@ var BottomCTA = React.createClass({
...
+      isFormDisabled = (this.state.step == this.STEP_WAIT_FOR_NETWORK);
```
Comment (ScottDowne) [inline pages/clubs.jsx]: ===

**Comment by toolness** [comment]
Just fixed those @ScottDowne, if the rest of this PR looks good to you, feel free to merge it!

**Pull request merged**